runtime/doc/news.txt

```diff
@@ -224,6 +224,8 @@ UI
 • Error messages are more concise:
   • "Error detected while processing:" changed to "Error in:".
   • "Error executing Lua:" changed to "Lua:".
+• |:restart| detaches the embedded server from the UI and then restarts
+  it.
```

Comment:
Before this gets into a larger release (or backported), would it make sense to say a bit more here? Just speaking for myself, I'm not sure that I would have picked up what this means for plugin development and use right away.